common/djangoapps/util/models.py

```diff
@@ -62,3 +63,49 @@ def to_python(self, value):
             value = decompress_string(value)
 
         return value
+
+
+class SerializedChoicesField(models.CharField):
```

Comment:
I put this class in this file because it seems to be something that could be more generally useful than just the application I'm using in this PR.